code/addons/interactions/package.json

```diff
@@ -82,7 +82,7 @@
     "@storybook/theming": "7.0.0-alpha.50",
     "@storybook/types": "7.0.0-alpha.50",
     "global": "^4.4.0",
-    "jest-mock": "^27.0.6",
+    "jest-mock": "^29.3.1",
```

Comment:
I don't think we can update this, since it's an actual dependency that is used by the interactions addon, and so far it only supports jest 27, AFAIK

Reply:
Aha, will downgrade, cc @yannbf